webrtc.html

```diff
@@ -5793,6 +5795,21 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
             <dd>
               <p>The number of channels (mono=1, stereo=2).</p>
             </dd>
+            <dt><dfn><code>maxptime</code></dfn> of type <span class=
+            "idlMemberType"><a>unsigned long</a></span></dt>
+            <dd>
+              <p>The maximum packetization time set on the
+              <code><a>RTCRtpSender</a></code>. Not specified if unset. If
```

Comment:
For an RTCRtpReceiver, should this be the maximum packetization time supported by the decoder?